src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -62,6 +65,10 @@
 
     @State
     @Nullable
+    ArrayList<Trigger> triggers;
```

Comment:
Usually it is good practice to use the interface on the declaration, unless you need that concrete type later (e.g. because it has a method that is not in the interface)